OrderCloud.Integrations.Payment.BlueSnap/BlueSnapService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using OrderCloud.Catalyst;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OrderCloud.Integrations.Payment.BlueSnap
+{
+	public class BlueSnapService : OCIntegrationService, ICreditCardSaver, ICreditCardProcessor
+	{
+		public BlueSnapService(BlueSnapConfig defaultConfig) : base(defaultConfig) { }
+
+		public async Task<string> GetIFrameCredentialAsync(OCIntegrationConfig overrideConfig = null)
```

Comment:
It is removed from the interface. This is BlueSnapService. I kept the name for this method, I still wanted to provide that functionality because people will need it, even if it's not part of the interface.